RCTComms/transport.py

```diff
@@ -401,3 +400,82 @@ def send(self, data: bytes, dest):
 
     def isOpen(self):
         return self.__socket is not None
+
+class RCTSerialTransport(RCTAbstractTransport):
+    '''
+    Serial Transport
+    socket.socket wrapped in a serial.Serial object wrapped in an abstract transport (move over, turducken)
+    '''
+    def __init__(self, port: str) -> None:
+        '''
+        Constructor for an RCTSerialTransport
+        :param port: port to be used in underlying socket connection
+        '''
+        self.__port = port
+        self.__serial: Optional[serial.Serial] = None
+
+    def open(self) -> None:
+        '''
+        Open the serial port.
+        '''
+        if self.__serial is None:
+            self.__serial = serial.Serial(self.__port)
+            self.__serial.set_buffer_size(rx_size=65536)
+        if not self.__serial.isOpen():
+            self.__serial.open()
+
+    def receive(self, buffer_len: int, timeout: int=None) -> Tuple[bytes, str]:
```

Comment:
Parameter 'bufLen' has been renamed to 'buffer_len' in overriding 'RCTSerialTransport.receive' methodPylint(W0237:arguments-renamed)

Reply:
Resolved with 662a21c
Replaced all usages of bufLen with buffer_len. Thanks for catching.